packages/design-tokens/src/figma/darkTheme.json

```diff
@@ -10,13 +10,25 @@
       "value": "{grey.1000}",
       "type": "color",
       "parent": "Theme Colors/Dark mode",
-      "description": "For sunken neutral surface below background/default."
+      "description": "For section bg usually over background/default."
```

Comment:
@Akatori-Design this is the description for `background/alternative` should it be
1. "For sunken neutral surface below background/default."
2. "For section bg usually over background/default."

Reply:
@georgewrmarshall We can keep description #1 for `background/alternative`. I intend to phase out of `background/alternative` by removing the option in Figma.

Reply:
Thanks! Updated here https://github.com/MetaMask/metamask-design-system/pull/682/commits/9f0d5c0c5bd44f448a1e73d1930e4f2d2d2a51de